.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
```

Comment:
i prefer semicolons. i also like single quotes in JS and double quotes in JSX/HTML. but idk if we can get that specific